src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -315,20 +326,22 @@ public void whenInvalidCursorsThenPreconditionFailed() {
                 .body("detail", equalTo("non existing partition very_wrong_partition"));
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 30000)
```

Comment:
Does this test really need 30 seconds as a timeout? I think that the value of this test is not enough for spending up to 30 seconds on it. Also I think it can be replaced by Unit Test.